connect/runtime/src/main/java/org/apache/kafka/connect/runtime/isolation/Plugins.java

```diff
@@ -235,6 +235,8 @@ public Converter newConverter(AbstractConfig config, String classPropertyName, C
         String configPrefix = classPropertyName + ".";
         Map<String, Object> converterConfig = config.originalsWithPrefix(configPrefix);
         plugin.configure(converterConfig, isKeyConverter);
+        log.debug("Configuring the {} converter with configuration:{}{}",
```

Comment:
I think this should be *above* the previous line? Order of output could be confusing given current phrasing. Might also want to include `isKeyConverter` in the log.

Reply:
Agree with moving. `isKeyConverter` is used in the first parameter in the log statement.